internal/cmd/beta/intake/instance/create/create.go

```diff
@@ -0,0 +1,249 @@
+	labelsFlag              = "labels"
+	catalogURIFlag          = "catalog-uri"
+	catalogWarehouseFlag    = "catalog-warehouse"
+	catalogNamespaceFlag    = "catalog-namespace"
+	catalogTableNameFlag    = "catalog-table-name"
+	catalogPartitioningFlag = "catalog-partitioning"
+	catalogPartitionByFlag  = "catalog-partition-by"
+	catalogAuthTypeFlag     = "catalog-auth-type"
+	dremioTokenEndpointFlag = "dremio-token-endpoint" //nolint:gosec // false positive
+	dremioPatFlag           = "dremio-pat"
+)
+
+// inputModel struct holds all the input parameters for the command
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	// Top-level fields
+	DisplayName *string
+	RunnerId    *string
+	Description *string
+	Labels      *map[string]string
+
+	// Catalog fields
+	CatalogURI          *string
+	CatalogWarehouse    *string
+	CatalogNamespace    *string
+	CatalogTableName    *string
+	CatalogPartitioning *string
+	CatalogPartitionBy  *[]string
+
+	// Auth fields
+	CatalogAuthType     *string
+	DremioTokenEndpoint *string
+	DremioToken         *string
+}
+
+func NewCmd(p *params.CmdParams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates a new Intake",
+		Long:  "Creates a new Intake.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a new Intake with required parameters`,
+				`$ stackit beta intake create --display-name my-intake --runner-id xxx --catalog-uri "http://dremio.example.com" --catalog-warehouse "my-warehouse"`),
+			examples.NewExample(
+				`Create a new Intake with a description, labels, and Dremio authentication`,
+				`$ stackit beta intake create --display-name my-intake --runner-id xxx --description "Production intake" --labels "env=prod,team=billing" --catalog-uri "http://dremio.example.com" --catalog-warehouse "my-warehouse" --catalog-auth-type "dremio" --dremio-token-endpoint "https://auth.dremio.cloud/oauth/token" --dremio-pat "MY_TOKEN"`),
+			examples.NewExample(
+				`Create a new Intake with manual partitioning by a date field`,
+				`$ stackit beta intake create --display-name my-partitioned-intake --runner-id xxx --catalog-uri "http://dremio.example.com" --catalog-warehouse "my-warehouse" --catalog-partitioning "manual" --catalog-partition-by "day(__intake_ts)"`),
+		),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p.Printer, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p.Printer, p.CliVersion)
+			if err != nil {
+				return err
+			}
+
+			projectLabel, err := projectname.GetProjectName(ctx, p.Printer, p.CliVersion, cmd)
+			if err != nil {
+				p.Printer.Debug(print.ErrorLevel, "get project name: %v", err)
+				projectLabel = model.ProjectId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create an Intake for project %q?", projectLabel)
```

Comment:
```suggestion
				prompt := fmt.Sprintf("Are you sure you want to create an Intake instance for project %q?", projectLabel)
```

In case it's an instance, I would phrase it like this